src/Persistence/Initialization/Version095d/Maps/Icarus.cs

```diff
@@ -30,6 +30,9 @@ public Icarus(IContext context, GameConfiguration gameConfiguration)
     /// <inheritdoc/>
     protected override string MapName => "Icarus";
 
+    /// <inheritdoc/>
+    protected override byte SafezoneMapNumber => 4;
```

Comment:
Please replace these "magic numbers" by constants.
For example, Lorencia or Devias have internal consts for their numbers. Add constants to LostTower and the other safezone maps as well, and use them instead of these "magic numbers".

Reply:
okay makes total sense i am sry <3.

Shouldnt this be checked by the code style analyes also? xD

Reply:
hmm i am wondering right now if should use const or static readonly. But i th ink it should be static readonly because it might be inizialised from config files or db or sth xD.

Reply:
I suggest "internal const" in that case, because these values never change ;-)
Usually, you use "static readonly" only for public "constants", or for stuff which can't be defined as a const.